library/iFixit/Matryoshka/Memcached.php

```diff
@@ -36,15 +36,14 @@ protected function __construct(\Memcached $memcached) {
    }
 
    public function set($key, $value, $expiration = 0) {
-      $setValue = $this->memcached->set($key, $value, $expiration);
-      $resultCode = $this->memcached->getResultCode();
-
-      if ($resultCode !== \Memcached::RES_SUCCESS) {
-         $errorMessage = $this->memcached->getResultMessage();
-         throw new \MemcachedException($errorMessage, $resultCode);
+      if ($this->memcached->set($key, $value, $expiration) {
```

Comment:
I am not sure how the linter didn't check this, but we are missing a closing `)` for the if conditional. As a result the webpage fails to load 
![Return token](https://user-images.githubusercontent.com/22064420/122837862-b604b580-d2a9-11eb-8dbe-e079b463406e.png)

Reply:
Ha, thanks!

Reply:
```suggestion
      if ($this->memcached->set($key, $value, $expiration)) {
```